### PR TITLE
Replace srext_ext_xslt_source with local constant

### LINKED ISSUE #86: change `srext_ext_xslt_source` to local constant
https://github.com/SAP/abap-file-formats-tools/blob/a0a7570b24dc0d64a8952994693bc8953c71d63e/src/zcl_aff_writer_xslt.clas.testclasses.abap#L936

add use in https://github.com/SAP/abap-file-formats-tools/blob/a0a7570b24dc0d64a8952994693bc8953c71d63e/src/zcl_aff_writer_xslt.clas.testclasses.abap#L934
plus 3 more occurrences

### PR DESCRIPTION
**Pull request by wurzka**
closes #86

### CODE REVIEW

File: src/zcl_aff_writer_xslt.clas.testclasses.abap [hunk 1]
```diff
@@ -933,7 +935,7 @@ CLASS ltcl_integration_test IMPLEMENTATION.
     DATA(st_name) = CONV progname( c_xslt_prefix && st_execution_counter ).
     st_name = |{ st_name WIDTH = 30 PAD = '=' }XT|.
```
Comment (larshp) [inline src/zcl_aff_writer_xslt.clas.testclasses.abap:936]: there is also this line, which I guess is the same constant?

File: src/zcl_aff_writer_xslt.clas.testclasses.abap [hunk 2]
```diff
@@ -3501,7 +3505,7 @@ CLASS ltcl_integration_test_ad IMPLEMENTATION.
...
```
Comment (larshp) [inline src/zcl_aff_writer_xslt.clas.testclasses.abap:3506]: same

File: src/zcl_aff_writer_xslt.clas.testclasses.abap (hunk 1 above)
Comment (wurzka) [reply src/zcl_aff_writer_xslt.clas.testclasses.abap:936]: Sorry, I don't see what you mean. This line concatenates the st name with some '=' und the letters 'XT'

Comment (larshp) [reply src/zcl_aff_writer_xslt.clas.testclasses.abap:936]: the newly defined constant contains "XT"..., can it be used instead of this hardcoded string?

Comment (wurzka) [reply src/zcl_aff_writer_xslt.clas.testclasses.abap:936]: Ah yeah, sure. Will be fixed in next commit

**Review by albertmink — APPROVED** [review]
Looks good to me.

**Pull request merged**